.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "recommendations": [
+      "IQGeo.iqgeo-utils-vscode",
+      "ms-vscode-remote.vscode-remote-extensionpack",
+      "ms-azuretools.vscode-docker",
+      "GitHub.vscode-pull-request-github",
+      "ms-vsliveshare.vsliveshare"
```

Comment:
This could be very useful. My thought is that if it does get included, it should be added to the `devcontainer.json` file so that it's installed when launching the dev container. There may be a few who develop without using the dev container extension or those performing the initial setup. However, most others should be using the dev container extension.

Reply:
can/should it be in both then?